CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+# v1.0.3 (2020-11-19)
+
+## OS Changes
+* Support setting Linux kernel parameters (sysctl) via settings ([#1158], [#1171])
+* Add `bpffs` mount to default installation ([#1160], **thanks @izaakschroeder!**)
+* Create links under `/dev/disk/ephemeral` for ephemeral storage devices ([#1173])
+* Package updates and patches ([#1176], [#1195], [#TBD])
```

Comment:
```suggestion
* Update third-party package dependencies ([#1176], [#1195])
* Backport containerd 1.4 fixes until we can update ([#TBD])
```
I don't think the containerd patches should be lumped with general dependency updates; if we're not able to update but feel these are important fixes, that's an unusual situation worth calling out.

Reply:
nit: also, it'd be too big a GitHub suggestion to be make directly, but can we move these to Build Changes instead of OS Changes?  We don't expect any impact to users, so that might feel more appropriate?  (It doesn't look like we've been consistent about this in the past, but I'd like us to have some standard.)